draft-ietf-mls-protocol.md

```diff
@@ -766,7 +766,9 @@ delete the leaf_secret.
 
 ## Synchronizing Views of the Tree
 
-The members of the group need to keep their views of the tree in
+After generating a Commit as described in the prior section, the generator of
```

Comment:
This stuff might be better phrased in terms of a `DirectPath` (which maybe we should rename to `UpdatePath`?), since a Commit no longer has to have an update of this character.

Reply:
`UpdatePath` sounds like a better name. I'll do this renaming in a separate PR, but will rephrase this section to refer to a `DirectPath` as opposed to a `Commit` in the meantime.

Reply:
Actually, you know, I think it would be better to do this change in a separate commit as well, as it will involve re-writing the "Ratchet Tree Evolution" section to refer to `DirectPath` as well, no?

Reply:
Given this and the discussion below, do you want to close this and come back after the other refactors are done?